artemis-server/src/main/java/org/apache/activemq/artemis/core/config/Configuration.java

```diff
@@ -1509,4 +1509,32 @@ default String resolvePropertiesSources(String propertiesFileUrl) {
 
    void setManagementRbacPrefix(String prefix);
 
+   /** This configures the Mirror Ack Manager number of attempts on queues before trying page acks.
+    *  It is not intended to be configured through the XML.
+    *  The default value here is 5. */
+   int getMirrorAckManagerMinQueueAttempts();
+
+   Configuration setMirrorAckManagerMinQueueAttempts(int minQueueAttempts);
+
+   /** This configures the Mirror Ack Manager number of attempts on page acks.
+    *  It is not intended to be configured through the XML.
+    *  The default value here is 2. */
+   int getMirrorAckManagerMaxPageAttempts();
+
+   Configuration setMirrorAckManagerMaxPageAttempts(int maxPageAttempts);
+
+   /** This configures the interval in which the Mirror AckManager will retry acks when
+    *  It is not intended to be configured through the XML.
+    *  The default value here is 100, and this is in milliseconds. */
+   int getMirrorAckManagerRetryDelay();
+
+   Configuration setMirrorAckManagerRetryDelay(int delay);
+
+   /** Should Mirror use Page Transactions When target destinations is paging?
+    *  When a target queue on the mirror is paged, the mirror will not record a page transaction for every message.
+    *  The default is false, and the overhead of paged messages will be smaller, but there is a possibility of eventual duplicates in case of interrupted communication between the mirror source and target.
+    *  If you set this to false there will be extra syncs and a record stored on the journal for the page-transaction additionally to the record in the page store. */
```

Comment:
The 'default is false, and the overhead will be smaller'....but then, 'if you set this false, there will be \<extra overhead\>' ?    Which is it?